templates/procuration/choose_election.html.twig

```diff
@@ -6,74 +6,45 @@
                         Attention : si vous n'êtes pas en France métropolitaine, votre date de vote peut varier.
                     </div>
                 </div>
-                <div class="text--body b__nudge--top-50">
-                    <h2>Comment ça marche ?</h2>
-                    <ol>
-                        <li>
-                            Cochez les élections qui vous concernent au-dessus. Et cliquez sur "Continuer".
-                        </li>
-                        {% if is_request %}
-                            <li>
-                                Remplissez le formulaire en indiquant vos coordonnées et votre bureau de vote.
-                            </li>
-                            <li>
-                                <em>En Marche !</em> recherche une personne de confiance dans votre commune disponible le jour du vote.
-                            </li>
-                            <li>
-                                Vous recevrez d'ici quelques jours toutes les informations pour compléter le formulaire CERFA avec les coordonnées de la personne qui votera en votre nom.
-                            </li>
-                            <li>
-                                Le jour de l'élection, un électeur <em>En Marche !</em> vote en votre nom.
-                            </li>
-                        {% else %}
-                            <li>
-                                Vous remplissez le formulaire en indiquant vos coordonnées et disponibilités.
-                            </li>
-                            <li>
-                                En cas de demande de procuration, vous êtes mis en relation avec les personnes
-                                absentes les jours de vote.
-                            </li>
-                            <li>
-                                Vous votez en leur nom.
-                            </li>
-                        {% endif %}
-                    </ol>
-                    <a href="{{ asset('en-marche_fiches-procurations.pdf') }}" target="_blank">En savoir plus &raquo;</a>
+                <div class="form__checkbox b__nudge--top b__nudge--bottom">
+                    {{ form_errors(elections_form.elections) }}
+                    {{ form_widget(elections_form.elections) }}
                 </div>
-            </div>
 
-            {{ form_widget(elections_form.choose, { label: 'Continuer' }) }}
+            </div>
+            <div>
+                {{ form_widget(elections_form.choose, { label: 'Continuer' }) }}
+            </div>
 
         {{ form_end(elections_form) }}
+
+        <div class="text--gray">
+            {{ include('procuration/legal.html.twig', { elections: elections_form.vars.elections|join(', ') }, with_context=false) }}
+        </div>
+
     </section>
 
-    {{ include('procuration/legal.html.twig', { elections: elections_form.vars.elections|join(', ') }, with_context=false) }}
```

Comment:
Be careful if you remove this, it is not included anywhere else for now.

Reply:
I didn't take it out, I just moved it!